packages/eas-cli/src/submissions/ArchiveSource.ts

```diff
@@ -163,8 +163,11 @@ async function handleBuildIdSourceAsync(source: ArchiveBuildIdSource): Promise<A
       source,
     };
   } catch (err) {
-    Log.error(chalk.bold(`Couldn't find build for id ${source.id}`));
-    Log.error(err);
+    Log.error(chalk.bold(`Could not find build with ID ${source.id}`));
+    Log.warn(
+      'Are you sure you did not pass the build ID from the legacy build service (expo build:(android|ios))?'
+    );
+    Log.warn('Only EAS Build build IDs are supported.');
```

Comment:
```suggestion
    Log.warn('Build IDs from the classic build service (expo build:[android|ios]) are not supported. Learn more.');
```

where "Learn more" is a terminal link to https://docs.expo.dev/submit/classic-builds/